Stripe/PublicHeaders/UI/STPPaymentCardTextField.h

```diff
@@ -0,0 +1,162 @@
+//
+//  STPPaymentCardTextField.h
+//  Stripe
+//
+//  Created by Jack Flintermann on 7/16/15.
+//  Copyright (c) 2015 Stripe, Inc. All rights reserved.
+//
+
+@import UIKit;
+
+@class STPPaymentCardTextField;
+
+/**
+ *  This protocol allows a delegate to be notified when a payment text field's contents change, which can in turn be used to take further actions depending on the validity of its contents.
+ */
+@protocol STPPaymentCardTextFieldDelegate <NSObject>
+@optional
+/**
+ *  Called when either the card number, expiration, or CVC changes. At this point, one can call -isValid on the text field to determine, for example, whether or not to enable a button to submit the form. Example:
+ 
+ - (void)paymentCardTextFieldDidChange:(STPPaymentCardTextField *)textField {
+      self.paymentButton.enabled = textField.isValid;
+ }
+ 
+ *
+ *  @param textField the text field that has changed
+ */
+- (void)paymentCardTextFieldDidChange:(nonnull STPPaymentCardTextField *)textField;
+
+@end
+
+
+/**
+ *  STPPaymentCardTextField is a text field with similar properties to UITextField, but specialized for collecting credit/debit card information. It manages multiple UITextFields under the hood to collect this information. It's designed to fit on a single line, and from a design perspective can be used anywhere a UITextField would be appropriate.
```

Comment:
nit: not sure if the implementation detail of `STPPaymentCardTextField` managing multiple `UITextFields` is necessary